torch/_prims_common/wrappers.py

```diff
@@ -320,12 +319,12 @@ def _fn(*args, out=None, **kwargs):
             sig.empty,
             out_type,
         )
-        params = chain(sig.parameters.values(), (out_param,))
+        params = *sig.parameters.values(), out_param
+        params = sorted(params, key=lambda p: p.kind)
```

Comment:
Why is this change necessary?

Reply:
Without that line, if there's a `**kwds` parameter, it's placed before the `out=` parameter, and you get an exception `ValueError: wrong parameter order: variadic keyword parameter before keyword-only parameter`.

I added a comment to this effect.